src/discussions/tours/data/selectors.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const notRespondedFilterTour = ({ tours }) => {
+  // This function filters the tours list in the state by the tour_name 'not_responded_filter'
+  // and returns the filtered list. This can be useful for displaying only the 'not_responded_filter'
+  // tours to the user, for example in a list or table.
+  if (!tours) {
+    return {};
+  }
```

Comment:
This case will never run because you are already find the redux state and redux state will not be undefined or null.